tests/test_summit/test_rm_create_on_localhost.py

```diff
@@ -53,6 +54,8 @@ def test_rm_create_on_localhost():
     import pprint
     pprint.pprint(lrms.lrms_info)
     assert lrms.lrms_info == {'agent_nodes'     : {},
+                              'gpus_per_node'   : 6,
+                              'cores_per_node'  : 20,
```

Comment:
Should this be 40?

Reply:
Good point - I think the test case still comes from summitdev.  The actual values don't matter though, as they are part of the test case.  As long as the values match, the method is doing the right thing.